src/utils/email/sendPaawordLink.js

```diff
@@ -0,0 +1,69 @@
+import nodemailer from 'nodemailer';
+import ejs from 'ejs';
+import path from 'path';
+
+class Email {
+    constructor (user, url) {
+        this.to = user.email;
+        this.firstName = user.firstName;
+        this.url = url;
+        this.from = `Barefoot Nomad <${process.env.EMAIL_FROM}>`;
+    }
+
+    newTransport() {
+        if (process.env.NODE_ENV === 'production') {
```

Comment:
```suggestion
        if (process.env.NODE_ENV !== 'test') {
```

So that when someone is testing it locally in the "development" env they can see the email being sent